ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -350,6 +350,92 @@ namespace
                                 scores_ps[0].get_length();
                 }
             }
+            return result;
+        }
+
+        std::vector<int64_t> get_integers(const std::shared_ptr<HostTensor>& input,
+                                          const Shape& shape)
+        {
+            size_t input_size = shape_size(shape);
+            std::vector<int64_t> result(input_size);
+
+            switch (input->get_element_type())
+            {
+            case element::Type_t::i8:
+            {
+                auto p = input->get_data_ptr<int8_t>();
+                for (size_t i = 0; i < input_size; ++i)
+                {
+                    result[i] = int64_t(p[i]);
+                }
+            }
+            case element::Type_t::i16:
```

Comment:
Missing `break` statement?

```suggestion
            }
            break;
            case element::Type_t::i16:
```